include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6662,7 +6662,57 @@ namespace VectorTools
                               norm, weight, exponent);
   }
 
+  template <int dim, int spacedim, class InVector>
+  double error_norm_from_cell_errors(const Triangulation<dim,spacedim> &tria,
+                                     const InVector &cellwise_error,
+                                     const NormType &norm)
+  {
+    Assert( cellwise_error.size() == tria.n_active_cells(),
+            ExcMessage("input vector cell_error has invalid size!"));
+
+    MPI_Comm comm = MPI_COMM_SELF;
+#ifdef DEAL_II_WITH_MPI
+    if (const parallel::Triangulation<dim,spacedim> *ptria =
+          dynamic_cast<const parallel::Triangulation<dim,spacedim>*>(&tria))
+      comm = ptria->get_communicator();
+#endif
+
+    switch (norm)
+      {
+      case L2_norm:
+      case H1_seminorm:
```

Comment:
Ok, i see what you mean. But I was considering this function to operate on errors without the underlying interpretation of where these errors come from. Whereas you and Timo think of it as a companion to `integrate_difference()`. 

I agree that in this context it makes sense and the documentation reflects their marriage already.